src/PrestaShopBundle/Resources/views/Admin/Module/Includes/card.html.twig

```diff
@@ -49,7 +50,9 @@
             {% endif %}
           </div>
           <div class="module-read-more-{{display_type}} small no-padding">
-            <a class="module-read-more-{{display_type}}-btn url" href="#" data-toggle="modal" data-target="#module-modal-read-more-{{module.attributes.name }}{{ additionalModalSuffix|default('') }}">{{ 'Read More'|trans({}, 'Admin.Modules.Feature') }}</a>
+            {% if module.attributes.id != "0" %}
```

Comment:
We don't have any informations about modules that have been installed using zip files: maybe there is a wording issue here ? ping @AlexEven